### DISCUSSION TITLE
Folding buttons redesign?

### DISCUSSION THREAD

**Discussion by rexelbartolome** [freeplane/freeplane#317] [Ideas]
This is a continuation of the discussion from SF talking about the folding buttons right beside the node which euu mentioned: https://sourceforge.net/p/freeplane/discussion/758437/thread/8b6f0779e8/?limit=25&page=1#6dfd in which I suggested for it to be more similar to Mindmeister's https://imgur.com/1BbVAGL

I think the plus symbology isn't consistent with the other ways of we present the concept of folding

So the folding circles currently look like this

<p align="center" width="100%">
current with hover <br>
    <img width="80%" src="https://user-images.githubusercontent.com/88552647/163657334-a786392c-1cd0-4b7c-8256-3330b2bfa85d.png">
</p>

<p align="center" width="100%">
current without hover <br>
    <img width="80%" src="https://user-images.githubusercontent.com/88552647/163657224-5df9fe92-618a-42f0-b07a-bf1831546d0a.png">
</p>

So I'd like to propose for it to look like this instead:
<p align="center" width="100%">
proposed look <br>
    <img width="80%" src="https://user-images.githubusercontent.com/88552647/163657227-ce160340-9975-40b3-9f01-c087e48f74fd.png">
</p>

This also brings me to an issue with borderless map styles (i.e., fully transparent node borders) since if the node border is transparent, it also erases the folding button entirely, not signifying whether or not a node has children or not. So I think it would be great if the visibility of the folding button is maintained and doesn't inherit the transparency of the node border but inherits the node edge instead (or maybe just have a forced alpha of 100)
<p align="center" width="100%">
current behavior with borderless map style <br>
    <img width="80%" src="https://user-images.githubusercontent.com/88552647/163657759-21f560aa-3f8a-48c6-a49e-dc8aa7b04f93.png">
</p>

<p align="center" width="100%">
proposed behavior with borderless map style <br>
    <img width="80%" src="https://user-images.githubusercontent.com/88552647/163657230-32e8ea21-1618-4dae-8248-7ffa18179153.png">
</p>

Also, I haven't asked @dpolivaev how easy can this be changed so I'd like to know your thoughts

**Comment by dpolivaev** [comment]
I tried to implement your suggestions. I decided to simplify them to reduce the visual complexity of the mind maps. The results is contained in the latest preview 1.10.1_06

**Reply by evlibra** [reply]
It would be good to have the option to disable "display folding buttons" - it is missing references (ion FreeMind it is at appearance ->"display folding buttons" ), or maybe someone can point out if it can be done by editing the config file?
-for me it is annoying to see that "bubble" popping up when I move mouse over the nodes 

(I'm switching to Freeplane from Freemind, and that is one of the most noticeable disadvantage for me, of course on top of ton of advantages Freeplane progressed with)

Earlier, the same was discussed here: https://sourceforge.net/p/freeplane/discussion/758437/thread/44c1a1cc/

Thanks

**Reply by dpolivaev** [reply]
Try
Preferences…->Appearance->Size limits->Folding mark width
Preferences…->Appearance->Size limits->Folding switch width

**Reply by quickfold** [reply]
There are minimum limits of 3 on the folding mark and folding switch width, so changing that isn't equivalent to disabling them.
